bodhi/server/templates/update.html

```diff
@@ -657,6 +693,12 @@ <h4>Reboot Required</h4>
             % endif
           </div>
           % endif
+          % if 'testing' in update.status or 'stable' in update.status:
```

Comment:
This will work for RPMs but not for Flatpaks or containers. I'm not sure about modules, but I think they might not work either because of https://github.com/fedora-infra/bodhi/issues/2805.

This is OK though, we don't need to support all types ever to merge this patch, but for now perhaps we should add a condition to the ```if``` statement to only show this for RPMBuild updates.

```suggestion
          % update.content_type.description == 'rpm' and ('testing' in update.status or 'stable' in update.status):
```

Reply:
I'll ask around for the right commands for flatpaks, containers and modules to see if I can make a one-for-all patch.
I've started used modules only recently and I don't have a great knowledge on them.